### PR TITLE
Moved vspec2franca to anytree

### PR DESCRIPTION
**Pull request by danielwilms**
- changed to anytree in creation
- adopted struct and generated output

@gunnarx, could you please check, if correct?

### CODE REVIEW

**Comment by erikbosch** [comment]
@danielwilms @SebastianSchildt noticed that both your builds fail. Do not know if it concerns some form of circular dependency, that we specify that we require vss-tools in Pipfile but there is no (recent) one.

File: vspec2x.py [hunk 1]
```diff
@@ -32,6 +33,7 @@ def export(config: argparse.Namespace, root: VSSNode):
     csv = vss2csv
     yaml = vss2yaml
     binary = vss2binary
+	franca = vss2franca
```
Comment (erikbosch) [inline vspec2x.py:36]: Odd indent, tab?

Comment (danielwilms) [reply vspec2x.py:36]: true, will change that

**Comment by danielwilms** [comment]
> @danielwilms @SebastianSchildt noticed that both your builds fail. Do not know if it concerns some form of circular dependency, that we specify that we require vss-tools in Pipfile but there is no (recent) one.

I should have mentioned it in the PR. I took away the version flag, which was specific for franca. Either we leave it, and I'll implement it or we take it away from the makefile as well. Let's discuss tonight.

**Comment by erikbosch** [comment]
Possibly need to rebase to get the new buildcheck changes

**Comment by erikbosch** [comment]
@danielwilms - do you think you have time to fix the review comments before the meeting tomorrow (and possibly rebase, if that helps to make the build pass)

**Review by erikbosch — APPROVED** [review]
Looks good for me

**Comment by SebastianSchildt** [comment]
Looking good to me "structure-wise", I can not  say anything about the output

**Comment by danielwilms** [comment]
decision to merge 2022/05/16

**Pull request merged**